scripts/check_file_license_apache.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+FLAG=$1
+FILE=$2
+SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
+TIMESCALE_LOCATION=`dirname $SCRIPTPATH`
+
+LICENSE_FILE=
+LICENSE_STRING=
+FIRST_COMMENT=
+
+if [[ "$FLAG" == '-c' ]]; then
```

Comment:
I think we've been slowly transitioning our "style" in bash to use `${FLAG}` instead of `$FLAG`. Would be nice if we could do that throughout this PR